nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/FeaturesRunner.java

```diff
@@ -175,13 +175,13 @@ protected void stop() throws Exception {
         loader.apply(Direction.BACKWARD, holder -> holder.feature.stop(FeaturesRunner.this));
     }
 
-    protected void beforeSetup() throws Exception {
-        loader.apply(Direction.FORWARD, holder -> holder.feature.beforeSetup(FeaturesRunner.this));
+    protected void beforeSetup(final FrameworkMethod method, final Object test) {
```

Comment:
I couldn't find any use for the Object test parameter (Same for `afterTeardown`) except passing it along ?

Reply:
I prefer to backport the same code than master which will ease future backports if we need it.